core/tests/puppeteer-acceptance-tests/specs/logged-out-user/click-all-buttons-on-navbar.spec.ts

```diff
@@ -49,7 +49,7 @@ describe('Logged-out User', function () {
     'should open About Foundation Url with The Oppia Foundation button ' +
       'in About Menu on navbar',
     async function () {
-      await loggedOutUser.clickAboutFoundationButtonInAboutMenuOnNavbar();
+      await loggedOutUser.clickAboutButtonInAboutMenuOnNavbar();
```

Comment:
Shouldn't this whole it block be removed? Since you are removing all code related to about foundation.